### PR TITLE
Worker references - Ethnicty, Nationality, Country of Birth, Qualifications and Recruited From

### PR DESCRIPTION
**Pull request by wozzer72**
These five stories all have reference data lookups; all reference data lookups done together for implementation efficiency. And their respective Worker properties.

Includes the table fact (e.g. CREATE TABLE) and table changes (ALTER TABLE); the latter only being relevant for DEV DB (sfcdevdb).

Ethnicity & Qualifications are the easy ones and we've seen them before (Main Job); it's just a lookup value; so the "Value" column name is prefixed with "FK".

With the other three reference based stories, we have the already seen value (e.g. "British", "Other", "Yes" or "No"), but now, if "Other", there is a drop down (lookup) against the topic reference. So a sixth column has been introduced to hold the "OtherFK".

Backend code written with new API endpoints for each new reference and automated integration tests updated to assert results. Data values have been verified by Maria.

Backend code for the additional properties is Work in Progress, but I would like to be able to get the DEV DB (sfcdevdb) updated so Denny can start work on the UI screens But I do expect by the time we walked through this that I'll have Ethnicity and Nationality (one of each type) available as evidence.

### CODE REVIEW

**Comment by wozzer72** [comment]
[Recruited from #152](https://trello.com/c/a01Sa1vO/68-recruited-from-152)

**Comment by wozzer72** [comment]
[Country of birth #152](https://trello.com/c/W7XnNxhH/66-country-of-birth-152)

**Comment by wozzer72** [comment]
[Highest level of Social Care qualification #152](https://trello.com/c/ACAfFG8D/81-highest-level-of-social-care-qualification-152)

File: worker-db-ddl.sql [hunk 1]
```diff
@@ -38,6 +47,585 @@ CREATE TYPE cqc."WorkerDisability" AS ENUM (
...
+	(15, 500, 'Black / African / Caribbean / Black British', 'African'),
+	(16, 501, 'Black / African / Caribbean / Black British', 'Caribbean'),
+	(17, 502, 'Black / African / Caribbean / Black British', 'Any other Black / African / Caribbean background'),
+	(18, 600, 'Other ethnic group', 'Arab'),
+	(19, 601, 'Other ethnic group', 'Any other ethnic group');
+
+-- Nationality Reference Data
+CREATE TABLE IF NOT EXISTS cqc."Nationality" (
+	"ID" INTEGER NOT NULL PRIMARY KEY,
+	"seq" INTEGER NOT NULL, 	-- this is the order in which the Ethinicity will appear without impacting on primary key (existing foreign keys)
```
Comment (Shakir-Sopra) [inline worker-db-ddl.sql]: Hi Warren
The Nationality table, columns seq and nationality can be "Seq" and "Nationality", though this isn't affecting functionality anyway however we need to follow the format, what you advise on this?
Thanks
Shakir

Comment (wozzer72) [reply worker-db-ddl.sql]: Good catch Shakir. I have updated the schema here, and updated the backend code to match the revised column names, and have retested all of the five reference APIs that consume these tables.

If you happy with the revised schema and feedback to this comment, please approve the PR and I'll look to merging down to master, and effecting the changes on the DEV database instance.

**Comment by Shakir-Sopra** [comment]
Confirming merge after review and approval.

Thanks
Shakir

**Comment by Shakir-Sopra** [comment]
Branch deleted safely.

Thanks
Shakir

**Pull request merged**